src/main/resources/data/minecolonies/loot_tables/recipes/netherworker/trip.json

```diff
@@ -0,0 +1,656 @@
+{
+    "name": "minecolonies:recipes/trip5",
```

Comment:
This name doesn't match the filename -- I assume this was because you had separate loot tables per hut level originally before adding the scoreboard stuff.  TBH, it'd be nicer if you went back to doing that, the JEI integration already supports separate loot tables per hut level but doesn't currently support scoreboard conditions.

Reply:
The separate loot tables never existed, they were planned but I got the scoreboard conditions working, and never went there. 

Datagen was broken for me all week, so I've been working around it, and now I'm running out of holiday time, so I'd like to take that as a potential follow-on change.

Reply:
Thank you for your help with this conversion.